src/components/NonProfitApply/NonProfitApply.js

```diff
@@ -44,25 +49,45 @@ export default function NonProfitApply() {
   const { user } = useAuth0();
   var image = "/npo_placeholder.png";
   var nonProfitOptions = [];
+  
+
+  const options = {
+    autoConfig: true, // set pixel's autoConfig. More info: https://developers.facebook.com/docs/facebook-pixel/advanced/
+    debug: false, // enable logs
+  };
+  var advancedMatching = null; // { em: 'some@email.com' }; // optional, more info: https://developers.facebook.com/docs/facebook-pixel/advanced/advanced-matching
+  if( user && user.email )
+  {
+    advancedMatching = { em: user.email };
+  }
+  ReactPixel.init(process.env.NEXT_PUBLIC_FACEBOOK_PIXEL_ID, advancedMatching, options);
 
+  useEffect(() => {
+    ReactRecaptcha3.init(process.env.NEXT_PUBLIC_GOOGLE_CAPTCHA_SITE_KEY).then(
+      (status) => {
+        // console.log(status); // Should log SUCCESS
+      }
+    );
+  }, []);
 
   const START_DATE = "Saturday, Oct 7th";
   const END_DATE = "Sunday, Oct 8th 2023";
   const LOCATION_ARIZONA = "Phoenix, Arizona (to be determined)"
   const LOCATION = LOCATION_ARIZONA + " and virtual";
 
-
-  nonprofits.forEach((item) => {
+  nonprofits && nonprofits.forEach((item) => {
```

Comment:
bugfix: If we don't get a nonprofit list from the backend, we just don't populate these as options in the form